examples/pytorch/ogc/ogc.py

```diff
@@ -0,0 +1,103 @@
+import time
+import argparse
+import scipy.sparse as sp
+
+import torch
+import torch.nn.functional as F
+
+from dgl import AddSelfLoop
+from dgl.data import CiteseerGraphDataset, CoraGraphDataset, PubmedGraphDataset
+from utils import sparse_mx_to_torch_sparse_tensor, symmetric_normalize_adjacency, LinearNeuralNetwork
+
+
+# Training settings
+decline = 0.9            # the dcline rate
+eta_sup = 0.001          # the learning rate for supervised loss
+eta_W = 0.5              # the learning rate for updating W
+beta = 0.1               # in [0,1], the moving probability that a node moves to its neighbors
+max_similar_tol = 0.995  # the max_tol test set label prediction similarity between two iterations
+max_patience = 2         # the tolreance for consecutively getting very similar test prediction
+
+
+def update_U(U, Y, predY, W):
+    global eta_sup
+    # ------ update the smoothness loss via LGC ------
```

Comment:
ditto. start with uppercase and end with dot.

Reply:
Thanks. We have adjusted all of them.